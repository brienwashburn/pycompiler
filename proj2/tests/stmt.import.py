import a
import a as b
import a.q
import a.q.f.g as c
import a, b, c
import a as foo, b as bar, c as rax
import a.bar as foo, b.foo as bar, c.rax as rax
