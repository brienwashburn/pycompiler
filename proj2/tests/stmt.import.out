(Module
 (Import (a #f))
 (Import (a b))
 (Import (a.q #f))
 (Import (a.q.f.g c))
 (Import (a #f) (b #f) (c #f))
 (Import (a foo) (b bar) (c rax))
 (Import (a.bar foo) (b.foo bar) (c.rax rax)))
